uiowa/uiowa
# Update Git commands.

Resolves #1736 by deleting all but the 5 newest tags from each remote.

Additionally adds a `blt uiowa:git:deploy` command to deploy the latest release to each application production environment. Tested with 2.8.9. Moved some code around to share between GitCommands and MultisiteCommands.

### Testing
- Run `blt uiowa:git:clean` and see that it will delete `refs/tags/2.8.4-build` from each remote. Confirm.
- Run `blt uiowa:git:deploy` and see that it would deploy the latest tag to each remote. Cancel.
